addons/website/static/src/js/menu/seo.js

```diff
@@ -433,106 +622,59 @@ var SeoConfigurator = Dialog.extend({
-            self.$('input[name=seo_page_keywords]').removeAttr('readonly').attr('placeholder', "");
-            self.$('button[data-action=add]').prop('disabled', false).removeClass('disabled');
-        });
-        this.keywordList.on('selected', self, function (word, language) {
-            self.keywordList.add(word, language);
-        });
-        this.keywordList.on('content-updated', self, function (removed) {
-            self.updateTable(removed);
-        });
-        this.keywordList.insertAfter(this.$('.table thead'));
-        this.disableUnsavableFields().then(function(){
-            self.renderPreview();
-            self.metaImageSelector = new metaImageSelector(self, {
+        this.disableUnsavableFields().then(function () {
+            // Image selector
+            self.metaImageSelector = new MetaImageSelector(self, {
                 title: self.htmlPage.getOgMeta().metaTitle,
+                description: self.htmlPage.getOgMeta().metaDescription,
                 metaImg : self.metaImg || self.htmlPage.getOgMeta().ogImageUrl,
                 pageImages : _.pluck(self.htmlPage.images().get(), 'src'),
             });
-            self.metaImageSelector.appendTo(self.$('.js_seo_keywords_list'));
-        });
+            self.metaImageSelector.appendTo(self.$('.js_seo_image'));
+
+            // title and description
+            self.metaTitleDescription = new MetaTitleDescription(self, {
+                canEditTitle: self.canEditTitle,
+                canEditDescription: self.canEditDescription,
+                isIndexed: self.isIndexed,
+                title: self.htmlPage.title(),
+                description: self.htmlPage.description(),
+                url: self.htmlPage.url(),
+            });
+            self.metaTitleDescription.on('title-changed', self, self.titleChanged);
+            self.metaTitleDescription.on('description-changed', self, self.descriptionChanged);
+            self.metaTitleDescription.appendTo(self.$('.js_seo_meta_title_description'));
 
-        this.getLanguages();
-        this.updateTable();
-    },
-    getLanguages: function () {
-        var self = this;
-        this._rpc({
-            model: 'website',
-            method: 'get_languages',
-            args: [[weContext.get().website_id]],
-            context: weContext.get(),
-        }).then( function (data) {
-            self.$('#language-box').html(core.qweb.render('Configurator.language_promote', {
-                'language': data,
-                'def_lang': weContext.get().lang
-            }));
+            // keywords
+            self.metaKeywords = new MetaKeywords(self, {htmlPage: self.htmlPage});
+            self.metaKeywords.appendTo(self.$('.js_seo_meta_keywords'));
         });
     },
     disableUnsavableFields: function () {
         var self = this;
         return this.loadMetaData().then(function (data) {
+            // no reload if not on page (condition: website_id === undefined)
```

Comment:
correct me if I am missing something but this can't happen?

Reply:
The comment is not the best but the condition is happening if we edit SEO of something else than a Page.